pynest/nest/lib/hl_api_types.py

```diff
@@ -331,7 +331,7 @@ def get(self, *params, **kwargs):
         set
         """
 
-        if self.empty():
+        if not self.__bool__():
```

Comment:
wouldn't just ``if not self`` work?
I don't know if there would be any significant cost to it (I don't think so) but it would look much nicer to me...

Reply:
23.3 µs for `if not self.__bool__()` vs 33.1 µs for `if not self`. So not really a significant performance hit, and it looks cleaner.